hudi-flink/src/main/java/org/apache/hudi/streamer/HoodieFlinkStreamer.java

```diff
@@ -89,7 +89,7 @@ public static void main(String[] args) throws Exception {
         .uid("uid_kafka_source")
         .map(new RowDataToHoodieFunction<>(rowType, conf), TypeInformation.of(HoodieRecord.class))
         // Key-by partition path, to avoid multiple subtasks write to a partition at the same time
```

Comment:
please update the description accordingly

Reply:
Thanks, done.